suitcase/tiff_series/tests/tests.py

```diff
@@ -62,9 +60,8 @@ def test_export(tmp_path, example_data, stack_images):
     '''
 
     collector = example_data()
-    artifacts = export(collector, tmp_path, file_prefix='',
-                       stack_images=stack_images)
-    expected = create_expected(collector, stack_images)
+    artifacts = export(collector, tmp_path, file_prefix='')
```

Comment:
This is not nesting any of the templating....

Reply:
can you provide more context @tacaswell I don't understand your statement.

Reply:
Nesting -> testing?

Reply:
Nesting -> testing?

Reply:
Ahhhh thanks @danielballan for the translation. I will add some tests on the templating.

Reply:
fixed

Reply:
Sorry, n and t are next to each other on my keyboard and spell check did not complain about 'nesting'....